### PR TITLE
Add Support For Smartcode 888 Deadbolt - 888.xml

### PR DESCRIPTION
**Pull request by mike240se**
Support for Smartcode 888 Deadbolt

### CODE REVIEW

**Comment by mike240se** [comment]
The 888 needs the same work around as the other Smartcode locks, it doesnt send a doorlock report when manually locked/unlocked.  The lock does not have the same features of the 914c (no dip switches) and is a more stripped down version of the lock (cheap) so the smartcode.xml wont work either, too many features not supported.

File: config/kwikset/888.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0090:0541:0003</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/kwikset/smartcode.png</MetaDataItem>
```
Comment (Fishwaldo) [inline config/kwikset/888.xml:4]: Is the pic really the same as smartcode?

File: config/kwikset/888.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Please only perform a factory reset when the primary controller is missing or inoperable.
+
+To perform a factory reset, please perform the following:
+1. Remove the battery pack
+2. Press and HOLD the Program button while reinserting the battery pack.
+3. Keep holding the button for 30 seconds until the lock beeps and the status LED flashes red.
+4. Press the Program button once more.  The status LED will flash green and red several times.
+5. After a few seconds, the lock will initiate the door handing process.</MetaDataItem>
+  </MetaData>
+  <!-- Configuration Parameters -->
```
Comment (Fishwaldo) [inline config/kwikset/888.xml:34]: There is no configuration values for this device?

File: config/kwikset/888.xml (hunk 1 above)
Comment (mike240se) [reply config/kwikset/888.xml]: no they are close but not exact, i put smartcode.png as a temp and forgot to go back and fix it.  i have updated the PR to include the image and updated the xml to use the correct image 200x200 png

File: config/kwikset/888.xml (hunk 2 above)
Comment (mike240se) [reply config/kwikset/888.xml:34]: there are no dip switches like the other kwikset.  The lock does have 2 configurable options: auto lock on off and delay, and mute/unmute audio but from what I can tell they are only configurable through button presses on the device.  Perhaps you can tell me in #zwave on HA discord how to confirm its not possible to access these settings via zwave if you dont mind?

Comment (mike240se) [reply config/kwikset/888.xml:34]: command classes in mqtt: https://i.imgur.com/DPdymA2.png

Comment (Fishwaldo) [reply config/kwikset/888.xml:34]: Usually the product manual (or engineering manual) will specify if there are any configuration paramaters available via Z-Wave. Anyway. I'll merge this one.

Comment (mike240se) [reply config/kwikset/888.xml:34]: thanks,  then there are no z-wave configurable options, i checked all the documentation.

Comment (MagmaiKH) [reply config/kwikset/888.xml:34]: This doc for the Kwikset 888 shows a number of parameters selectable by z-wave.
https://www.cd-jackson.com/zwave_device_uploads/864/Install-Guide-and-Manual-888-ENG.pdf

See § 2
> 2.  Configuration Parameters The Z-Wave door lock module supports the use of the configuration command class to provide advanced configuration of the door lock over the Z-Wave network.  This section describes the configuration parameters supported by the door lock.

| Param  | Wosit |
| ------------- | ------------- |
| 33/34 | SKU |
| 35 | Int. LED en/dis |
| 36 | Auto Lock en/dis|
| 37 | Audio en/dis|
| 40 | Factory Reset |
| 47 | Motor Overload Alarm en/dis|

(I can't even get it to lock or unlock yet though.)

Comment (mike240se) [reply config/kwikset/888.xml:34]: Thank you, I used the doc you provided to get them added.

As far as lock/unlock, the antenna in the 888 seems quite poor, i had bad results until I added an extender near my lock.

**Pull request merged**